packages/pds/src/account-manager/index.ts

```
@@ -187,7 +189,8 @@ export class AccountManager
   }
 
   async updateRepoRoot(did: string, cid: CID, rev: string) {
-    return repo.updateRoot(this.db, did, cid, rev)
+    await repo.updateRoot(this.db, did, cid, rev)
+    await this.repoRevCache?.set(did, rev)
```

may be good to fail open on this rather than returning an error to the user

---

oh rip saw you handle these errors in the simple store 😛